### ISSUE TITLE
A11y_OSPO Production_Open Source Updates_Link Purpose: NVDA/Narrator announces the entire tile as a description of the links present under "Open Source" section.

### ISSUE THREAD

**Issue by Rachita98** [microsoft/opensource.microsoft.com#123]
<h2> GitHub Tags:</h2>
#A11yTCS;#A11yUsable;#Win10;#DesktopWeb;#ChromiumEgde;#A11ySev4;#P2_WCP;#WCP;#P2_OSPOProduction_Web_Sep2021;#P2_OSPOProduction;#OSPO Production;#NVDA;#Narrator;

<h2> Environment Details:</h2>
URL: https://opensource.microsoft.com/

<h2> OS and Browser Details:</h2>
Microsoft Windows 10 Enterprise Version 21H1 (OS build 19043.1165)
Microsoft Edge Version 95.0.1000.0 (Official build) dev (64-bit)

<h3> Note:</h3>
This is a high impact usability bug and could have a big negative impact on the user’s experience.

<h2> Repro Steps:</h2>
1. Hit the URL:https://opensource.microsoft.com/
2. Tab till the links present under "Open Source" section.
3. Check whether the entire tile is being read as a description of the links present under "Open Source" section or not.

<h2>Actual Result:</h2>
NVDA/Narrator announces the entire tile as a description of the links present under "Open Source" section.

<h2> Expected Result:</h2>
NVDA/Narrator should not announce the entire tile as a description of the links present under "Open Source" section.

<h2>User Impact:</h2>
Users with low vision may not understand the content clearly if,  the entire tile is being read as a description of the links present under "Open Source" section.

<h2>Attachments for reference:</h2>
**Narrator Behaviour:**

https://user-images.githubusercontent.com/90394513/133634336-ad1ac361-aa60-44bd-b270-22d3bfef7647.mp4

**NVDA Behaviour:**
![#123_Link Purpose_NVDA announces the entire tile as a description of the links present under Open Source section](https://user-images.githubusercontent.com/90394513/133633508-0c0d7d67-79a3-4348-8e22-529a0e008e33.png)

**Comment by jeffwilcox** [comment]
This should get reviewed. However, I do believe that "open source updates" section has been removed...

**Comment by chanya2534** [comment]
> This should get reviewed. However, I do believe that "open source updates" section has been removed...

**Comment by chanya2534** [comment]
> This issue has been identified as stale because it has gone 14 days with no activity.
> The issue will be closed in 5 days. If this is incorrect, simply comment on the issue, or remove the stale label.

**Comment by jeffwilcox** [comment]
Confirming this section was removed.
